index.php

```diff
@@ -7,6 +7,35 @@
 // Creates a variable which points to the correct database and gives username and password
 $db = new PDO('mysql:host=db;dbname=dinosaurs;', 'root', 'password');
 
+/**
+ * Checks if the form has been submitted and if so returns the dinos from the submitted search. If not, returns all dinos
+ *
+ * @param PDO $db database
+ * @return string of dinos to be displayed
+ */
+function checkIfSearched(PDO $db): string {
+    if (isset($_GET['submit'])) {
+        return Museum::displayAllDinos(DinosaurHydrator::getSearchedDinos($db, $_GET['search'], $_GET['filter']));
```

Comment:
may be dryer to put each hydrator equal to an $output variable then return Museum::DisplayAllDinos($output)

Reply:
Going to leave for now and see what happens in trainer review, as discussed as they aren't currently calling the same function so are pretty dry and I think this may add unnecessary complexity